zcash_client_sqlite/CHANGELOG.md

```diff
@@ -42,6 +42,9 @@ and this library adheres to Rust's notion of
   - `FsBlockDbError::CacheMiss`
 - `zcash_client_sqlite::FsBlockDb::write_block_metadata` now overwrites any
   existing metadata entries that have the same height as a new entry.
+- The `v_transactions` and `v_tx_outputs` views now return the 32-byte
+  transaction identifier for transactions instead of the internal database
+  id.
```

Comment:
`v_transactions` already returned the txid; the change there is only to remove `id_tx`.

Additionally, was the bug that this PR is fixing always present in `v_transactions`? I suspect so (and that users just weren't seeing it before because the mobile SDKs were still using direct DB access to provide history), in which case there should also be an entry under `### Fixed`.

Reply:
Yup, I think that's correct.